embedding/csharp/Tizen.Flutter.Embedding/Interop/Interop.cs

```diff
@@ -140,6 +140,16 @@ public struct FlutterDesktopEngineProperties
             uint scan_code,
             [MarshalAs(UnmanagedType.U1)]
             bool is_down);
+
+        [DllImport("flutter_tizen.so")]
+        public static extern void FlutterDesktopViewSetFocused(
+                            FlutterDesktopView view,
```

Comment:
Please follow the [formatting convention](https://github.com/flutter-tizen/flutter-tizen/pull/426).

https://github.com/flutter-tizen/flutter-tizen/pull/444/files#diff-225e3c79a0ee53a2065e2001017139ae15e14f249305f64622597e00c3a40f0d

Reply:
Thanks. done

Reply:
Thanks. Please also consider fixing the `FlutterDesktopViewOnKeyEvent` part.

Reply:
Also `focus` should be either `focused` or `is_focused`.

Reply:
> Please also consider fixing the `FlutterDesktopViewOnKeyEvent` part.

Indentation...